website/members/migrations/0038_auto_20200513_2143.py

```diff
@@ -0,0 +1,17 @@
+# Generated by Django 3.0.2 on 2020-05-13 19:43
```

Comment:
Where is this migration coming from?

Reply:
https://github.com/svthalia/concrexit/pull/1109/files?file-filters%5B%5D=.mo&file-filters%5B%5D=.py#diff-612fa3582bd5e7670c6bbddc784428eaL97

Reply:
removal of the permission from the member model